ConcernsCaseWork/ConcernsCaseWork/Pages/Team/SelectColleagues.cshtml.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Ardalis.GuardClauses;
+using ConcernsCaseWork.Extensions;
+using ConcernsCaseWork.Pages.Base;
+using ConcernsCaseWork.Security;
+using ConcernsCaseWork.Services.Teams;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.Pages.Team
+{
+	[Authorize]
+	[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+	public class SelectColleaguesPageModel : AbstractPageModel
+	{
+		private readonly IRbacManager _rbacManager;
+		private readonly ILogger<SelectColleaguesPageModel> _logger;
+		private readonly ITeamsModelService _teamsService;
+
+		[BindProperty]
+		public IList<string> SelectedColleagues { get; set; }
+		public string[] Users { get; set; }
+
+		public SelectColleaguesPageModel(IRbacManager rbacManager, ILogger<SelectColleaguesPageModel> logger, ITeamsModelService teamsService)
+		{
+			_rbacManager = Guard.Against.Null(rbacManager);
+			_logger = Guard.Against.Null(logger);
+			_teamsService = Guard.Against.Null(teamsService);
+		}
+
+		public async Task<ActionResult> OnGetAsync()
+		{
+
+			try
+			{
+				_logger.LogMethodEntered();
+				await LoadPage();
+			}
+			catch (Exception ex)
+			{
+				_logger.LogErrorMsg(ex);
+
+				TempData["Error.Message"] = ErrorOnGetPage;
+			}
+			return Page();
+		}
+
+		public async Task<ActionResult> OnPostSelectColleagues()
+		{
+			try
+			{
+				_logger.LogMethodEntered();
+
+				Guard.Against.Null(SelectedColleagues);
+
+				await _teamsService.UpdateCaseworkTeam(new Models.Teams.ConcernsTeamCaseworkModel(_CurrentUserName, SelectedColleagues.ToArray()));
+
+				// Todo, go back to home page, with team tab selected.
```

Comment:
What's our position on 'TODO's'? Not sure if we're happy to allow them into production code (I'm fine with whatever we agree, so just checking).

Reply:
removed the comment, it was literally done on the next line